docs/07-AdvancedUsage.md

```diff
@@ -249,7 +249,7 @@ You can also use the `@dataprovider` annotation for creating dynamic examples, u
     /**
      * @return array
      */
-    protected function pageProvider() // alternatively, if you want the function to be public, be sure to prefix it with `_`
+    public function pageProvider() // alternatively, if you want the function to be public, be sure to prefix it with `_`
```

Comment:
Your change conflicts with comment.

Reply:
@Naktibalda i'm not sure what really comment means. Remove it as outdated

Reply:
If you make method public in Cest file, Codeception will run it as a test.

Reply:
Wow, I see.
My mistake is create `Test` file instead of `Cest`.

Sorry for any inconvenience